src/main/java/org/jenkinsci/plugins/ghprb/GhprbPullRequest.java

```diff
@@ -449,14 +449,18 @@ private void checkComment(GHIssueComment comment) throws IOException {
         // return;
         // }
 
-        if (helper.isWhitelistPhrase(body) && helper.isAdmin(sender)) { // add to whitelist
-            GHIssue parent = comment.getParent();
-            GHUser author = parent.getUser();
-            if (!helper.isWhitelisted(author)) {
-                logger.log(Level.FINEST, "Author {0} not whitelisted, adding to whitelist.", author);
-                helper.addWhitelist(author.getLogin());
+        if (helper.isWhitelistPhrase(body)) { // add to whitelist
+            if (helper.isAdmin(sender)) { // check if request is from an admin
+                GHIssue parent = comment.getParent();
+                GHUser author = parent.getUser();
+                if (!helper.isWhitelisted(author)) {
+                    logger.log(Level.FINEST, "Author {0} not whitelisted, adding to whitelist.", author);
+                    helper.addWhitelist(author.getLogin());
+                }
+                setAccepted(true);
+            } else {
+                logger.log(Level.FINEST, "Sender {0} is not an admin, rejecting whitelist request for {1}.", new Object[] { sender.getName(), author });
```

Comment:
Could you set the logging level to `INFO` for this message? I think that (or `WARNING`) is the better level for the failure condition.